components/LiveCard.js

```diff
@@ -1,44 +1,102 @@
 import React from 'react';
-import TextStyles from '../constants/TextStyles';
+import { View, StyleSheet } from 'react-native';
 import Svg, { Circle } from 'react-native-svg';
-import { View, Text, StyleSheet } from 'react-native';
+import Animated from 'react-native-reanimated';
 import Colors from '../constants/Colors';
+import Text from '../components/Text';
+
+const {
+  block,
+  Clock,
+  clockRunning,
+  cond,
+  set,
+  spring,
+  startClock,
+  stopClock,
+  Value,
+} = Animated;
+
+function runSpring(clock, value, dest) {
+  const state = {
+    finished: new Value(0),
+    velocity: new Value(0),
+    position: new Value(0),
+    time: new Value(0),
+  };
+
+  const config = {
+    toValue: new Value(0),
+    damping: 3,
+    mass: 2,
+    stiffness: 100,
+    overshootClamping: false,
+    restSpeedThreshold: 0.1,
+    restDisplacementThreshold: 0.1,
+  };
+
+  return block([
+    cond(clockRunning(clock), 0, [
+      set(state.finished, 0),
+      set(state.time, 0),
+      set(state.position, value),
+      set(state.velocity, 4),
+      set(config.toValue, dest),
+      startClock(clock),
+    ]),
+    spring(clock, state, config),
+    cond(state.finished, [
+      // loop the animation
+      stopClock(clock),
+      set(state.finished, 0),
+      set(state.time, 0),
+      set(state.position, value),
+      set(state.velocity, 4),
+      set(config.toValue, dest),
+      startClock(clock),
+    ]),
+    state.position,
+  ]);
+}
+
+function AnimatedCircle() {
+  const clock = new Clock();
+  const scale = runSpring(clock, 1, 1);
+
+  return (
+    <Animated.View style={{ transform: [{ scale }] }}>
+      <Svg style={styles.circle} viewBox="0 0 100 100">
+        <Circle cx="50" cy="50" r="50" fill={Colors.red} fillOpacity="1" />
+      </Svg>
+    </Animated.View>
+  );
+}
 
 export default ({ style }) => {
   return (
     <View style={[style, styles.container]}>
-      <View style={{ flex: 1 }} />
       <Text style={styles.liveText}>WATCH LIVE</Text>
-      <Svg style={styles.circle} viewBox="0 0 100 100">
-        <Circle
-          cx="50"
-          cy="50"
-          r="50"
-          stroke="red"
-          strokeWidth="1"
-          fill="red"
-        />
-      </Svg>
-      <View style={{ flex: 1 }} />
+      <AnimatedCircle />
     </View>
   );
 };
 
 const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: Colors.darkestGray,
+  },
   liveText: {
-    ...TextStyles.subtitle,
+    marginRight: 16,
```

Comment:
Why don't we want to use the TextStyles thing?

Reply:
cause we have a `<Text/>` component for handling text 😅 
we should really use that `<Text/>` component for text stuff. and if we want to have a defined type system, then we can build something like a `<Header>` or something  🤔 

we should add props to the `<Text>` component for defined sizing probably

Reply:
Like we talked about, I kind of like the idea of wrapping all of these into
```js
<TitleText/>
<SubtitleText/>
<BodyText/>
```
or something of the sort. Maybe a "later" thing, but I like it.

Reply:
yah! let's put that as a TODO